src/app.tsx

```diff
@@ -1,22 +1,69 @@
-import * as React from 'react';
-import { ButtonContainer } from './components/button-container';
-import { FormComponent } from './components/form-component';
+import * as classNames from 'classnames'
+import * as React from 'react'
+import './app.css'
+import * as API from './business/api'
+import { Bio } from './components/bio'
+import { ColumnLayout } from './components/column-layout'
+import { Contact } from './components/contact'
+import { FeaturedProjects } from './components/featured-projects'
+import { Footer } from './components/footer'
+import { Header } from './components/header'
+import { SiteInfo } from './model/model'
+
+interface State {
+    siteInfo?: SiteInfo
+}
+
+class App extends React.Component<{}, State> {
+    constructor(props: {}) {
+        super(props)
+        this.state = {}
+    }
 
-class App extends React.Component<{}, {}> {
     render() {
-        return (
-            <div>
-                <h1>This is the app component.</h1>
-                <FormComponent formName="Controlled Input Example">
-                    <p>
-                        Changing the color of the buttons in the component below
-                        should not case this text input to lose its value.
-                    </p>
-                    <ButtonContainer />
-                </FormComponent>
-            </div>
-        );
+        if (this.state.siteInfo) {
+            return this.renderLoaded(this.state.siteInfo)
```

Comment:
glad to see you switched to just making sure this is truthy, instead of an actual bool